xrfeitoria/object/object_utils.py

```diff
@@ -247,19 +271,6 @@ class ObjectUtilsBlender(ObjectUtilsBase):
     # ------- Getter ------- #
     ##########################
 
-    @classmethod
-    def get_dimensions(cls, name: str) -> 'Vector':
-        """Get dimensions of the object.
-
-        Args:
-            name (str): Name of the object.
-
-        Returns:
-            Vector: Dimensions of the object.
-        """
-        cls.validate_name(name)
-        return cls._get_dimensions_in_engine(name=name)
-
     @classmethod
     def get_bbox(cls, name: str) -> 'Dict[Tuple[Vector, Vector]]':
```

Comment:
`get_bbox` and `get_bound_box` seem to be duplicated, as well as `_get_bbox_in_engine` and `_get_bound_box_in_engine`.

Reply:
fixed in ee5504d